src/main/java/com/gw2auth/oauth2/server/service/gw2account/apitoken/Gw2AccountApiTokenServiceImpl.java

```diff
@@ -169,13 +174,18 @@ public void checkTokenValidity() {
             }
 
             if (isValidState != null) {
-                updateEntities.add(new Gw2AccountApiTokenValidUpdateEntity(apiTokenEntity.accountId(), apiTokenEntity.gw2AccountId(), isValidState));
+                validUpdateEntities.add(new Gw2AccountApiTokenValidUpdateEntity(apiTokenValidCheckEntity.accountId(), apiTokenValidCheckEntity.gw2AccountId(), isValidState));
             }
         }
 
-        if (!updateEntities.isEmpty()) {
-            this.gw2AccountApiTokenRepository.updateApiTokensValid(now, updateEntities);
+        if (!validUpdateEntities.isEmpty()) {
+            this.gw2AccountApiTokenRepository.updateApiTokensValid(now, validUpdateEntities);
             LOG.info("updated API-Token validity for {} API-Tokens; valid={} invalid={} unknown={}", tokensToCheck.size(), counts[0], counts[1], counts[2]);
         }
+
+        if (!nameUpdateEntities.isEmpty()) {
+            this.gw2AccountRepository.updateGw2AccountNames(nameUpdateEntities);
+            LOG.info("updated GW2 Account names for {} accounts", counts[3]);
```

Comment:
Nit: These two log messages seem to be the only ones starting with lowercase characters. This is admittedly annoying me more than it should.

Reply:
its the other way around: most start with lowercase, only some with uppercase. Fixed that.